freqtrade/rpc/telegram.py

```diff
@@ -421,9 +426,19 @@ def _profit(self, update: Update, context: CallbackContext) -> None:
         stake_cur = self._config['stake_currency']
         fiat_disp_cur = self._config.get('fiat_display_currency', '')
 
+        start_date = datetime.fromtimestamp(0)
+        try:
+            if context.args:
+                timescale = int(context.args[0])
+                today_start = datetime.combine(date.today(), datetime.min.time())
+                start_date = today_start - timedelta(days=timescale)
```

Comment:
I've changed the custom keyboard validations slightly, so `/profit 5` is also valid (this implicitly also supports `/daily 10` ..).


The only point i'm not certain about is the message content itself.

It's now sending "First Trade Opened: 4 days ago" ... but in reality, it might be better to say "starting from 4 day ago" ... as it's not the first trade - but the first trade that was opened since the period we specify - so maybe, if start-date is specified, we might be using a different text instead.

Reply:
> The only point i'm not certain about is the message content itself.

Do you have any other text in mind?

Reply:
not really (nothing specific, anyway)
maybe something along the lines of "Profit since xxx" ... but any wording will work i guess